.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
-# orbs:
-#   wp-svn: studiopress/wp-svn@0.1
+orbs:
+  wp-svn: studiopress/wp-svn@0.1
```

Comment:
I know this is just uncommenting what's here, but do we need to bump the version?

I noticed in the plugin that I was going to reinstate, it has a version of `0.2` -- do you know what the latest version is?

https://github.com/studiopress/genesis-connect-woocommerce/blob/42d9ab484e11117e7a64f6ebf7f718cdb4293d9e/.circleci/config.yml#L4

Reply:
It looks like the latest is the 0.2.x line, but I don't think there's any functionality that is really needed for the `genesis-beta-tester` plugin:

https://github.com/studiopress/wp-svn-orb/releases